packages/core/src/common/_mixins.scss

```diff
@@ -103,6 +103,39 @@ $pt-dark-intent-text-colors: (
   @include force-hardware-acceleration();
 }
 
+// A collection of mixins that utilize the CSS Flexbox model to standardize
+// the layout and composition of Blueprint components.
+
+// pt-flex-parent styles
+@mixin pt-flex-parent() {
+  display: inline-flex;
```

Comment:
probably not enough to put `inline-flex` everywhere... block elements are quite useful.

Reply:
@giladgray fair, only `Button` uses `inline-flex`.